zos_concepts/templates/copy_template/README.md

```diff
@@ -0,0 +1,68 @@
+# Use Jinja templates with zos_copy
+This playbook demonstrates how to use local Jinja templates to copy
+datasets and files in z/OS using Red Hat Ansible Certified Content for IBM Z.
+
+This playbook uses:
+  - collection:
+    - ibm.ibm_zos_core
+  - modules:
+    - zos_copy
+    - zos_gather_facts
+    - zos_data_set
+
+It is a good practice to review the playbook contents before executing
+them. It will help you understand the requirements in terms of space, location,
+names, authority, and the artifacts that will be created and cleaned up.
+
+## Playbook Requirements
+This playbook requires:
+
+- [IBM® z/OS® core collection 1.7.0 or later](https://galaxy.ansible.com/ibm/ibm_zos_core)
+- [Ansible® 2.13 or later](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html)
+
```

Comment:
I think this is a good place to explain that the `unixy` callback is required to be on the controller. I'm thinking something like:

- [Community General Ansible Collection](https://github.com/ansible-collections/community.general)
The ansible.cfg file here contains a setting `stdout_callback = unixy` which changes the format of the playbook output. It has been set to better demonstrate the effects of whitespace (as compared to the default output formatting). The `unixy` callback is part of the `community.general` package (which is maintained by the Ansible community). The callback can be installed from Ansible Galaxy with the command:
 ```ansible-galaxy collection install community.general```
Alternatively, the `stdout_callback = unixy` line can be removed (or commented out) from the `ansible.cfg` file. The playbook will run the same, only the formatting of the playbook output, especially of whitespace, will be less apparent.

Reply:
Good point, I'll add it